### PR TITLE
[4.0] Action logs filters

### PR DESCRIPTION
**Pull request by brianteeman**
Make sure that all the filters have a valid label

### CODE REVIEW

File: administrator/components/com_actionlogs/forms/filter_actionlogs.xml [hunk 1]
```diff
@@ -10,8 +10,7 @@
 		<field
 			name="extension"
 			type="extension"
-			label="COM_ACTIONLOGS_EXTENSION"
-			description="COM_ACTIONLOGS_EXTENSION_FILTER_DESC"
```
Comment (Quy) [inline administrator/components/com_actionlogs/forms/filter_actionlogs.xml:14]: Remove language string.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1291934652b54dcd07ab505d1a5b2217439ed1c0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25825">issues.joomla.org/tracker/joomla-cms/25825</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 1291934652b54dcd07ab505d1a5b2217439ed1c0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25825">issues.joomla.org/tracker/joomla-cms/25825</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25825">issues.joomla.org/tracker/joomla-cms/25825</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**